web/unit_tests/resources/keyboards/test_deadkeys.js

```diff
@@ -34,284 +34,376 @@ function Keyboard_test_deadkeys()
   this.KM=0;
   this.KBVER="1.0";
   this.KMBM=modCodes.SHIFT /* 0x0010 */;
-  this.s_deadnums="..........";
+  this.s_deadnums=[{d:0},{d:1},{d:2},{d:3},{d:4},{d:5},{d:6},{d:7},{d:8},{d:9}];
```

Comment:
Can we go a step further for debug compilation? I'd like to see `this.d_0=0; ... this.d_a=10; ...` in the top part of the file, then reference deadkeys accordingly:

```
this.s_deadnums=[{d:this.d_0},...];
```

This helps the developer to track their deadkeys much better. Obviously, for the non-debug compile, just use integers as you already are.

Reply:
While that would be lovely to do, it turns out that the deadkey name information isn't available to the Web compiler at this time; the relevant field (`fk.dpDeadKeyArray`) is `nil` and does not contain name data for the deadkeys.  I'll mark it as an issue for Developer enhancement.